integration/spark/src/main/scala/org/apache/carbondata/spark/rdd/CarbonDataRDDFactory.scala

```diff
@@ -719,16 +720,50 @@ object CarbonDataRDDFactory {
               loadMetadataDetails.setLoadStatus(CarbonCommonConstants.STORE_LOADSTATUS_SUCCESS)
               val rddIteratorKey = CarbonCommonConstants.RDDUTIL_UPDATE_KEY +
                                    UUID.randomUUID().toString
+              if (useKettle) {
+                try {
```

Comment:
move `try` to `CarbonDataLoadForUpdate.run` only, we should limit the try scope, do the same for next `try` also.